packages/eslint-config/configs/react.js

```diff
@@ -11,6 +11,8 @@ module.exports = {
   },
   plugins: ['react-hooks'],
   rules: {
+    // Remove when https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/pull/757 gets released
+    'jsx-a11y/no-onchange': 'off',
```

Comment:
iirc eslint will complain with a non existing rule...

Reply:
The rule still exists, just got deprecated from the `recommended` config.

Reply:
I meant when they delete the rule, will find out... not sure we needed the content 😅